.changeset/eight-experts-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/create-app': patch
+---
+
+Add missing `yarn clean` for app
```

Comment:
Just one minor note - when we change templates, it means that existing users will be deviating from the new norm. Therefore it's helpful to write explicitly in the changelog entry what the user needs to to to stay up to date. So it would be nice if you just added a few lines along the lines of "Add the following to your root `package.json` to..." or so.

Reply:
I was just doing that. Let me push.

Reply:
Done @freben, is this sufficient?